2021-08-06 21:55:45 error: ER_BAD_FIELD_ERROR: Unknown column 'email' in 'where clause' 
Stack trace: 
#1 at Query.Sequence._packetToError (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
#2 at Query.ErrorPacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
#3 at Protocol._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:291:23)
#4 at Parser._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:433:10)
#5 at Parser.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:43:10)
#6 at Protocol.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:38:16)
#7 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:88:28)
#8 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:526:10)
#9 at Socket.emit (events.js:315:20)
#10 at addChunk (internal/streams/readable.js:309:12)
#11 --------------------
#12 at Pool.query (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Pool.js:199:23)
#13 at internal/util.js:297:30
#14 at new Promise (<anonymous>)
#15 at Pool.query (internal/util.js:296:12)
#16 at User.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:528:57)
#17 at step (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:48:23)
#18 at Object.next (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:29:53)
#19 at fulfilled (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:20:58)
#20 at processTicksAndRejections (internal/process/task_queues.js:93:5)

2021-08-06 21:55:47 error: ER_BAD_FIELD_ERROR: Unknown column 'email' in 'where clause' 
Stack trace: 
#1 at Query.Sequence._packetToError (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
#2 at Query.ErrorPacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
#3 at Protocol._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:291:23)
#4 at Parser._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:433:10)
#5 at Parser.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:43:10)
#6 at Protocol.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:38:16)
#7 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:88:28)
#8 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:526:10)
#9 at Socket.emit (events.js:315:20)
#10 at addChunk (internal/streams/readable.js:309:12)
#11 --------------------
#12 at Pool.query (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Pool.js:199:23)
#13 at internal/util.js:297:30
#14 at new Promise (<anonymous>)
#15 at Pool.query (internal/util.js:296:12)
#16 at User.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:528:57)
#17 at step (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:48:23)
#18 at Object.next (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:29:53)
#19 at fulfilled (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:20:58)
#20 at processTicksAndRejections (internal/process/task_queues.js:93:5)

2021-08-06 21:56:19 error: ER_BAD_FIELD_ERROR: Unknown column 'email' in 'where clause' 
Stack trace: 
#1 at Query.Sequence._packetToError (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
#2 at Query.ErrorPacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
#3 at Protocol._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:291:23)
#4 at Parser._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:433:10)
#5 at Parser.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:43:10)
#6 at Protocol.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:38:16)
#7 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:88:28)
#8 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:526:10)
#9 at Socket.emit (events.js:315:20)
#10 at addChunk (internal/streams/readable.js:309:12)
#11 --------------------
#12 at Pool.query (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Pool.js:199:23)
#13 at internal/util.js:297:30
#14 at new Promise (<anonymous>)
#15 at Pool.query (internal/util.js:296:12)
#16 at User.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:528:57)
#17 at step (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:48:23)
#18 at Object.next (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:29:53)
#19 at fulfilled (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:20:58)
#20 at processTicksAndRejections (internal/process/task_queues.js:93:5)

2021-08-06 21:56:29 error: ER_BAD_FIELD_ERROR: Unknown column 'email' in 'where clause' 
Stack trace: 
#1 at Query.Sequence._packetToError (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
#2 at Query.ErrorPacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
#3 at Protocol._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:291:23)
#4 at Parser._parsePacket (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:433:10)
#5 at Parser.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Parser.js:43:10)
#6 at Protocol.write (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\protocol\Protocol.js:38:16)
#7 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:88:28)
#8 at Socket.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Connection.js:526:10)
#9 at Socket.emit (events.js:315:20)
#10 at addChunk (internal/streams/readable.js:309:12)
#11 --------------------
#12 at Pool.query (E:\my-project\cloud\cloud-express+react\back\node_modules\mysql\lib\Pool.js:199:23)
#13 at internal/util.js:297:30
#14 at new Promise (<anonymous>)
#15 at Pool.query (internal/util.js:296:12)
#16 at User.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:528:57)
#17 at step (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:48:23)
#18 at Object.next (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:29:53)
#19 at fulfilled (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:20:58)
#20 at processTicksAndRejections (internal/process/task_queues.js:93:5)

2021-08-06 21:57:20 error: ER_NO_SUCH_TABLE: Table 'reactdb.oauth_access_tokens' doesn't exist 
Stack trace: 
#1 at OauthAccessToken.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:134:31)
#2 at step (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:48:23)
#3 at Object.throw (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:29:53)
#4 at rejected (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:21:65)
#5 at processTicksAndRejections (internal/process/task_queues.js:93:5)

2021-08-06 21:58:44 error: ER_NO_SUCH_TABLE: Table 'reactdb.oauth_access_tokens' doesn't exist 
Stack trace: 
#1 at OauthAccessToken.<anonymous> (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:134:31)
#2 at step (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:48:23)
#3 at Object.throw (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:29:53)
#4 at rejected (E:\my-project\cloud\cloud-express+react\back\node_modules\tspace-sqlorm\dist\lib\orm\Base.js:21:65)
#5 at processTicksAndRejections (internal/process/task_queues.js:93:5)
